styled/hamburger.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components/native';
+import { NavButton } from '../components/BaseComponents';
+import { TouchableOpacity } from 'react-native';
+
+//TODO: Fix top to match figma once we figure out iPhone alignment
+export const HamburgerButton = styled(NavButton)`
+  top: 63px;
+  left: 16px;
```

Comment:
I think you can solve this by moving `<Hamburger navigation={this.props.navigation} />` into the `MapView`, right above `SearchBar`. Then you can make the `top` here the same as the top margin on `SearchBar` (20px I believe), and it should stay lined up with the height of the search bar, which has mostly been looking fine on different kinds of screens.